### DISCUSSION TITLE
Testing Framework

### DISCUSSION THREAD

**Discussion by seraph776** [codewars/codewars.com#2789] [Q&A]
**Question**: Is the `unittesting` framework available for testing `Katas`?  If so, how do you enable it? 
I know it's available for testing `Kumites`; however, I do not see it for testing `Katas` ~ Thanks for any help.

**Comment by Oahaijil** [comment]
On coding challenge platforms like Codewars, the testing framework is built-in and does not need to be enabled separately. You only need to write your solution in the code editor and then click the "Test" or "Submit" button. The system will automatically run the preset test cases and display the results. If you don't see the test options, please check whether you have correctly entered the solution page for the corresponding Kata (challenge).
